x-team/auto-ui
# Fix glitches render for some search form fields

**Release Type:** *Fixes* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2475

## Description

*In both of view modes: List and Tabular, Select List, Select Field, and Status fields have some white background parts showing at the top, and they aren't vertically aligned with surrounding fields.*



## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Related PRs

*N/A*


## Steps to Test or Reproduce

1- Go to Admin page
2- Choose tabular view
3- The fields in the search section should be aligned and without any glitches.
4- Confirm the same for the list view.



## Impacted Areas in Application

- Search form



![search_form](https://user-images.githubusercontent.com/5392202/48423098-57ace900-e736-11e8-8d08-bc7f9a23727e.png)

## Gif Image

![fields_elements_aligned](https://user-images.githubusercontent.com/5392202/48423108-5bd90680-e736-11e8-8a4d-e3433ed9aaf5.gif)
